src/pdn/test/four_sides/pad.lef

```diff
@@ -0,0 +1,403 @@
+        RECT 16.755 75.635 17.445 80 ;
+        RECT 19.355 75.635 20.045 80 ;
+        RECT 21.955 75.635 22.645 80 ;
+        RECT 24.555 75.635 25.245 80 ;
+        RECT 27.155 75.635 27.845 80 ;
+        RECT 29.755 75.635 30.445 80 ;
+        RECT 32.355 75.635 33.045 80 ;
+        RECT 34.955 75.635 35.645 80 ;
+        RECT 37.555 75.635 38.245 80 ;
+        RECT 40.155 75.635 40.845 80 ;
+        RECT 42.755 75.635 43.445 80 ;
+        RECT 45.355 75.635 46.045 80 ;
+        RECT 47.955 75.635 48.645 80 ;
+        RECT 3.28 77.9 50.735 80 ;
+        RECT 50.555 75.635 51.245 78.86 ;
+      LAYER metal2 ;
+        RECT 3.755 75.635 4.445 80 ;
+        RECT 6.355 75.635 7.045 80 ;
+        RECT 8.955 75.635 9.645 80 ;
+        RECT 11.555 75.635 12.245 80 ;
+        RECT 14.155 75.635 14.845 80 ;
+        RECT 16.755 75.635 17.445 80 ;
+        RECT 19.355 75.635 20.045 80 ;
+        RECT 21.955 75.635 22.645 80 ;
+        RECT 24.555 75.635 25.245 80 ;
+        RECT 27.155 75.635 27.845 80 ;
+        RECT 29.755 75.635 30.445 80 ;
+        RECT 32.355 75.635 33.045 80 ;
+        RECT 34.955 75.635 35.645 80 ;
+        RECT 37.555 75.635 38.245 80 ;
+        RECT 40.155 75.635 40.845 80 ;
+        RECT 42.755 75.635 43.445 80 ;
+        RECT 45.355 75.635 46.045 80 ;
+        RECT 47.955 75.635 48.645 80 ;
+        RECT 3.28 77.9 50.735 80 ;
+        RECT 50.555 75.635 51.245 78.86 ;
+      LAYER metal3 ;
+        RECT 3.755 75.635 4.445 80 ;
+        RECT 6.355 75.635 7.045 80 ;
+        RECT 8.955 75.635 9.645 80 ;
+        RECT 11.555 75.635 12.245 80 ;
+        RECT 14.155 75.635 14.845 80 ;
+        RECT 16.755 75.635 17.445 80 ;
+        RECT 19.355 75.635 20.045 80 ;
+        RECT 21.955 75.635 22.645 80 ;
+        RECT 24.555 75.635 25.245 80 ;
+        RECT 27.155 75.635 27.845 80 ;
+        RECT 29.755 75.635 30.445 80 ;
+        RECT 32.355 75.635 33.045 80 ;
+        RECT 34.955 75.635 35.645 80 ;
+        RECT 37.555 75.635 38.245 80 ;
+        RECT 40.155 75.635 40.845 80 ;
+        RECT 42.755 75.635 43.445 80 ;
+        RECT 45.355 75.635 46.045 80 ;
+        RECT 47.955 75.635 48.645 80 ;
+        RECT 3.28 77.9 50.735 80 ;
+        RECT 50.555 75.635 51.245 78.86 ;
+      LAYER metal4 ;
+        RECT 3.755 75.635 4.445 80 ;
+        RECT 6.355 75.635 7.045 80 ;
+        RECT 8.955 75.635 9.645 80 ;
+        RECT 11.555 75.635 12.245 80 ;
+        RECT 14.155 75.635 14.845 80 ;
+        RECT 16.755 75.635 17.445 80 ;
+        RECT 19.355 75.635 20.045 80 ;
+        RECT 21.955 75.635 22.645 80 ;
+        RECT 24.555 75.635 25.245 80 ;
+        RECT 27.155 75.635 27.845 80 ;
+        RECT 29.755 75.635 30.445 80 ;
+        RECT 32.355 75.635 33.045 80 ;
+        RECT 34.955 75.635 35.645 80 ;
+        RECT 37.555 75.635 38.245 80 ;
+        RECT 40.155 75.635 40.845 80 ;
+        RECT 42.755 75.635 43.445 80 ;
+        RECT 45.355 75.635 46.045 80 ;
+        RECT 47.955 75.635 48.645 80 ;
+        RECT 3.28 77.9 50.735 80 ;
```

Comment:
This pin shape overlaps all the previously defined pin shapes in the same layer. Seems like the smaller pin shapes should be removed.